mon-pix/app/controllers/campaigns/restricted/join.js

```diff
@@ -6,8 +6,9 @@ import Controller from '@ember/controller';
 import ENV from 'mon-pix/config/environment';
 
 export default class JoinRestrictedCampaignController extends Controller {
-  queryParams = ['participantExternalId'];
+  queryParams = ['participantExternalId', 'student'];
```

Comment:
```suggestion
  queryParams = ['participantExternalId', 'user'];
```
Jusque là, les étudiant sont présents dans les inscriptions.
Le fait qu'un utilisateur soit lié à un étudiant est le but de la réconciliation.
Du coup, comme on est avant la réconciliation, ce terme d'étudiant est ambigu => schooling_registration ou user ?

Reply:
Depuis le changement du queryParam dans l'URL pour un stockage dans le localStorage ce commentaire n'est plus d'actualité. Par ailleurs, le token s'appelle désormais `externalUser`.